libdnf/module/modulemd/ModuleDependencies.cpp

```diff
@@ -0,0 +1,52 @@
+#include <modulemd/modulemd-simpleset.h>
+#include <sstream>
+
+#include "libdnf/utils/utils.hpp"
+#include "ModuleDependencies.hpp"
+
+ModuleDependencies::ModuleDependencies(ModulemdDependencies *dependencies)
+        : dependencies(dependencies)
+{}
+
+std::vector <std::map<std::string, std::vector<std::string> > > ModuleDependencies::getBuildRequires()
+{
+    auto cBuildRequires = modulemd_dependencies_peek_buildrequires(dependencies);
+    return getRequirements(cBuildRequires);
+
+}
+
+std::vector <std::map<std::string, std::vector<std::string> > > ModuleDependencies::getRequires()
+{
+    auto cRequires = modulemd_dependencies_peek_requires(dependencies);
+    return getRequirements(cRequires);
+}
+
+std::map<std::string, std::vector<std::string>> ModuleDependencies::wrapModuleDependencies(const void *moduleName, const void *streams) const
+{
+    std::map<std::string, std::vector<std::string> > moduleRequirements;
+
+    auto name = std::string((char *) moduleName);
```

Comment:
do not remove `const` qualifier,  use C++ style cast, 
static_cast<const char *>(moduleName)